react/components/ReturnTableModal.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+import { FormattedCurrency } from 'vtex.format-currency'
+import { Modal } from 'vtex.styleguide'
+
+import styles from '../styles.css'
+
+type ReturnTableModalProps = {
+  selectedRequestProducts: any[]
+  handleModalToggle: () => void
+  isModalOpen: boolean
+}
+const ReturnTableModal = ({
+  selectedRequestProducts,
+  isModalOpen,
+  handleModalToggle,
+}: ReturnTableModalProps) => {
+  return (
+    <Modal centered isOpen={isModalOpen} onClose={handleModalToggle}>
+      <div className="dark-gray">
+        {selectedRequestProducts.length ? (
+          <table className={`${styles.table} ${styles.tableModal}`}>
```

Comment:
It's better if we use the `useCSSHandle` hook to apply the classes. Doing the way we are doing here, if someone in the future deletes the class in the css file, it will appear as `undefined` in the DOM.